app/views/shared/_email_header.html.erb

```diff
@@ -0,0 +1,9 @@
+<table class="header">
+  <tr>
+    <td>
+      <%= link_to root_url, class: 'header-logo' do %>
+        <%= image_tag 'logo-black.png', class: 'header-logo__image' %>
```

Comment:
I ran into issues getting image_tags and stylesheet_link_tags to work properly with roadie locally. After testing different action_mailer.asset_hosts configurations, I used base64 as a temporary workaround to preview images locally for development. More info on similar bugs encountered here: https://github.com/Mange/roadie-rails/issues/10